packages/cozy-client/src/models/contact.js

```diff
@@ -134,22 +135,21 @@ export const getDisplayName = contact =>
  * @returns {string} - the contact's 'byFamilyNameGivenNameEmailCozyUrl' index
  */
 export const getIndexByFamilyNameGivenNameEmailCozyUrl = contact => {
-  const indexByFamilyNameGivenNameEmailCozyUrl = capitalize(
-    [
-      get(contact, 'name.familyName', ''),
-      get(contact, 'name.givenName', ''),
-      getPrimaryEmail(contact),
-      getPrimaryCozyDomain(contact)
-    ]
-      .join('')
-      .trim()
-  )
+  const indexByFamilyNameGivenNameEmailCozyUrl = [
+    get(contact, 'name.familyName', ''),
+    get(contact, 'name.givenName', ''),
+    getPrimaryEmail(contact),
+    getPrimaryCozyDomain(contact)
+  ]
+    .join('')
+    .trim()
+    .toLowerCase()
 
   return get(
     contact,
     'indexes.byFamilyNameGivenNameEmailCozyUrl',
     indexByFamilyNameGivenNameEmailCozyUrl.length === 0
-      ? {}
+      ? null
```

Comment:
Since we have a few cozies with this {} format, we should test if indexByFamilyNameGivenNameEmailCozyUrl === {} and set it to null if needed?